spec/classes/web_spec.rb

```diff
@@ -117,7 +116,8 @@ def package_provider_for_gems
           database_name: 'zabbix-server',
           database_user: 'zabbix-server',
           database_password: 'zabbix-server',
-          zabbix_server: 'localhost')
+          zabbix_server: 'localhost'
```

Comment:
@igalic: rubocop doesn't want the comma:
https://travis-ci.org/voxpupuli/puppet-zabbix/jobs/129740215

rubocop is here a bit stricter than in other modules. remove the cop or the commit? I vote for the commit.